# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rar2hashcat/target.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/Makefile.orig	2023-08-08 23:39:00.000000000 +0300
+++ b/Makefile	2025-04-12 06:59:58.474274379 +0300
@@ -13,7 +13,7 @@
 # List of source files
 SOURCES = $(TARGET).c base64_convert.c
 
-all: $(TARGET_LINUX) $(TARGET_WIN)
+all: $(TARGET_LINUX)
 
 # Build target for Linux
 $(TARGET_LINUX): $(SOURCES)
@@ -24,4 +24,4 @@
 	$(CC_WIN) $(CFLAGS) $^ -o $@
 
 clean:
-	rm -f $(TARGET_LINUX) $(TARGET_WIN)
\ No newline at end of file
+	rm -f $(TARGET_LINUX) $(TARGET_WIN)

